.github/workflows/net-perf-gke.yaml

```diff
@@ -157,42 +157,29 @@ jobs:
       - name: Set Environment Variables
         uses: ./.github/actions/set-env-variables
 
+      - name: Get Cilium's default values
+        id: default_vars
+        uses: ./.github/actions/helm-default
+        with:
+          image-tag: ${{ inputs.SHA || github.sha }}
+          chart-dir: ./untrusted/install/kubernetes/cilium
+
       - name: Set up job variables
         id: vars
         run: |
           if [ "${{ github.event_name }}" = "workflow_dispatch" ] || [ ${{ github.event.pull_request }} ] ; then
-            SHA="${{ inputs.SHA }}"
             OWNER="${{ inputs.PR-number }}"
+            SHA="${{ inputs.SHA }}"
           else
-            SHA="${{ github.sha }}"
             OWNER="${{ github.ref_name }}"
             OWNER="${OWNER//[.\/]/-}"
+            SHA="${{ github.sha }}"
           fi
 
-          CILIUM_INSTALL_DEFAULTS="--chart-directory=install/kubernetes/cilium \
-            --helm-set=cluster.name=${{ env.clusterName }}-${{ matrix.index }} \
+          CILIUM_INSTALL_DEFAULTS="--helm-set=cluster.name=${{ env.clusterName }}-${{ matrix.index }} \
```

Comment:
The values from `default_vars` are not added here.

Reply:
Oops fixed